scripts/segmentation_HALO-20240827a.md

```diff
@@ -0,0 +1,562 @@
+)
+
+seg2 = (
+    slice("2024-08-27T10:29:27", "2024-08-27T10:55:11"),
+    ["straight_leg"],
+    "ferry to EC track",
+    ["irregularity: spike in roll angle at 10:39:42 by +4.5 degree"]
+)
+
+seg3 = (
+    slice("2024-08-27T10:56:25", "2024-08-27T11:04:24"),
+    ["straight_leg", "ec_track"],
+    "southward EC track",
+)
+
+seg4 = (
+    slice("2024-08-27T11:04:24", "2024-08-27T11:06:27"),
+    ["straight_leg", "ascent", "ec_track"],
+    "southward EC track",
+)
+
+seg5 = (
+    slice("2024-08-27T11:06:27", "2024-08-27T11:18:02"),
+    ["straight_leg", "ec_track"],
+    "southward EC track",
+)
+
+seg6 = (
+    slice("2024-08-27T11:18:59", "2024-08-27T11:26:12"),
+    ["straight_leg"],
+    "departure from EC track - leg 1",
+)
+
+seg7 = (
+    slice("2024-08-27T11:26:33", "2024-08-27T11:28:34"),
+    ["straight_leg"],
+    "departure from EC track - leg 2",
+)
+
+seg8 = (
+    slice("2024-08-27T11:31:26", "2024-08-27T11:33:30"),
+    ["straight_leg"],
+    "departure from EC track - leg 3",
+)
+
+seg9 = (
+    slice("2024-08-27T11:33:58", "2024-08-27T11:35:56"),
+    ["straight_leg"],
+    "departure from EC track - leg 4",
+)
+
+seg10 = (
+    slice("2024-08-27T11:38:41", "2024-08-27T11:56:44"),
+    ["straight_leg", "ec_track"],
+    "continued southward EC track", #any coordination? Why sonde?
+    ["additional sonde dropped at 11:40:32"]
+)
+
+seg11 = (
+    slice("2024-08-27T11:59:38", "2024-08-27T12:56:05"),
+    ["circle"],
+    "counterclockwise southern circle",
+)
+
+seg11_1 = (
+    slice("2024-08-27T12:59:25", "2024-08-27T13:03:17"),#05:41"),
+    ["straight_leg", "ascent", "ec_track"],
+    "climbing higher on northward EC track",
+)
+seg11_2 = (
+    slice("2024-08-27T13:03:17", "2024-08-27T13:05:41"),
+    ["straight_leg", "ec_track"],
+    "northward EC track towards middle circle",
+)
+
+seg12 = (
+    slice("2024-08-27T13:14:37", "2024-08-27T14:14:09"),
+    ["circle"],
+    "counterclockwise middle circle",
+    ["irregularity: due to turbulences at 13:36:46-13:38:08 and 14:02:03-14:03:21"]
+)
+
+seg13 = (
+    slice("2024-08-27T14:17:22", "2024-08-27T14:37:22"),
+    ["straight_leg", "ec_track"],
+    "northward leg through middle circle",
+    ["start of BAHAMAS measurement gap: 14:37:22"]
+)
+
+seg14 = (
+    slice("2024-08-27T14:50:53", "2024-08-27T15:50:27"),
+    ["circle"],
+    "counterclockwise nothern circle",
+    ["end of BAHAMAS measurement gap: 14:57:20"]
+)
+
+seg15 = (
+    slice("2024-08-27T15:59:45", "2024-08-27T16:16:37"),
+    ["straight leg", "ec_track"],
+    "leg through northern circle",
+    ["contains EC overpass"]
+)
+# 1st crossing over Meteor
+seg16 = (
+    slice("2024-08-27T16:21:25", "2024-08-27T17:03:06"),
+    ["straight_leg", "meteor_coordination"],
+    "leg 1 overpassing METEOR",
+)
+
+seg17 = (
+    slice("2024-08-27T17:03:16", "2024-08-27T17:06:15"),
+    ["kind"],
+    "steep turn",
+    ["steep turn with constant roll angle, perhaps worth a new segment type `turn`?"]
+)
```

Comment:
I would not make this and similar patterns a segment as I cannot think of any use case for analysing it.